src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -378,12 +378,14 @@ public void testSnapshotBuilder() throws Exception {
     populateOmopTable();
 
     // Test getConcepts
-    var conceptResponse = dataRepoFixtures.getConcepts(steward, datasetId, 2);
-    List<String> conceptNames =
-        conceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
-    assertThat("Correct number of concepts are returned", conceptNames.size(), equalTo(2));
-    assertThat(
-        "expected concepts are returned", conceptNames, containsInAnyOrder("concept1", "concept3"));
+    var getConceptResponse = dataRepoFixtures.getConcepts(steward, datasetId, 2);
```

Comment:
This change seems unrelated, although I think its fine to leave. Am I missing something there?

Reply:
You are not missing something it is unrelated! that is residue from this PR being a completely different change (adding limit support for azure and adding an integration test). I'll revert it